packages/manager/apps/restricted/src/restricted/ErrorJumbotron/ErrorJumbotron.tsx

```diff
@@ -1,11 +1,14 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { useTranslation } from 'react-i18next';
 
 import jumbotron from './error-jumbotron.png';
 import './error-jumbotron.styles.scss';
+import Context from '@/context';
 
 const ErrorJumbotron = (): JSX.Element => {
   const { t } = useTranslation('restricted');
+  const { ovhSubsidiary } = useContext(Context);
+  const isNotUsSubsidiary = ovhSubsidiary !== 'US';
```

Comment:
User is navigated to restricted app if they dont have access to `/me` API as per the ticket. But here, you're depending on its response. So, you can check the Region from the environment variables instead of subsidiary imo.

Reply:
Done